sklearn/model_selection/tests/test_split.py

```diff
@@ -379,6 +373,110 @@ def test_stratifiedkfold_balance():
             assert_equal(np.sum(sizes), i)
 
 
+def test_binnedstratifiedkfold_balance():
+    for i in range(11, 17):
+        n_folds = 2 + int(10*np.random.rand())
+        y = np.random.randn(i)
+        np.random.shuffle(y)
+        sizes = []
+
+        cv = BinnedStratifiedKFold(n_folds,
+                  shuffle=False, random_state=None)
+        bskf = cv.split(y)
+
+        bins = np.array([np.percentile(y, q) for q in range(n_folds)])
+        for  train_index, test_index in bskf:
+            sizes.append(
+                        len(test_index)
+                        )
+        assert_true((np.max(sizes) - np.min(sizes)) <= 1)
+        assert_equal(np.sum(sizes), i)
+
+
+def test_binnedstratifiedkfold_bin_spacing():
+    "check if the binned `y` falls into bins of equal size (+/- 1)"
```

Comment:
Can you make all the test  docstrings to comments instead. (See https://github.com/scikit-learn/scikit-learn/pull/4432#issue-63362524)